tests/cli/test_run.py

```diff
@@ -150,3 +154,31 @@ def test_zt_version(capsys, options):
     assert lines == expected
 
     assert captured.err == ""
+
+
+@pytest.mark.parametrize('option, autohide, no_autohide', [
+        ('--autohide', True, False),
+        ('--no-autohide', False, True),
+        ('--debug', False, False),
+        ]
+)
+def test_autohide_option(capsys, option, autohide, no_autohide):
+    args = parse_args([option])
+    assert args.autohide == autohide
+    assert args.no_autohide == no_autohide
```

Comment:
I was thinking with the testing that we might be able to check that these options get passed through, rather than testing at this level - but the code perhaps isn't really structured for that right now.

Reply:
Hmm, yes it is possible to do that, but it'll get a little messy. I'll have to call `main` and check `stdout`. But the problem is `main` won't exit on it's own. So i'll have to use some kind of manual interrupt to kill main.

Reply:
To be clear, I think this approach is fine for now.